Carbon.Windows/Functions/Uninstall-CEventLog.ps1

```diff
@@ -0,0 +1,31 @@
+
+function Uninstall-CEventLog
+{
+    <#
+    .SYNOPSIS
+    Removes an event log.
+
+    .DESCRIPTION
+    The `Uninstall-CEventLog` function removes an event log from the local computer.

```

Comment:
Reword to "The `Uninstall-CEventLog` function removes an event log from the local computer, if it exists. If the event log doesn't exist, the function does nothing."